weekly-updates/project-lead-meetings/11_4_18.md

```diff
@@ -50,6 +53,8 @@ _INTERVIEW_QUESTION_READY_:
 [//]: # (don't actually put the question here)
 
 _TO DO_:
+- archiecture
+- begin coding
```

Comment:
If you're recruiting, come up with a question. You'd like peeps who know basic web dev, right?